content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
```

Comment:
```suggestion
The OpenTelemetry Collector Builder (OCB) is a command-line tool that
```

I know that the tool itself is named `ocb`, but the acronym should be in uppercase. When referring to the tool, it is ok to use lowercase.